ui-core/src/types/landingPageConfig.ts

```diff
@@ -52,9 +52,11 @@ export type NavbarItem =
   | NavbarItemMailingList
   | NavbarItemExternal
 
-type BaseNavBarItem = {
+export type NavBarItemType = 'INTERNAL' | 'INTERNAL_ANCHOR' | 'MAILING_LIST' | 'EXTERNAL'
+
+export type BaseNavBarItem = {
   id?: string
-  itemType: 'INTERNAL' | 'INTERNAL_ANCHOR' | 'MAILING_LIST' | 'EXTERNAL'
+  itemType: NavBarItemType
   text: string
   itemOrder: number
```

Comment:
do we also want control over whether or not it opens in a new page?

Reply:
that's a good idea, I'll defer that for a later ticket